editor/resources/meta.edn

```diff
@@ -720,6 +720,10 @@
    :label "ProGuard config",
    :default "",
    :path ["android" "proguard"]}
+  {:type :boolean,
+   :help "This attribute indicates whether the package installer extracts native libraries from the APK to the file system",
+   :default true,
+   :path ["android" "extract_native_libs"]}
```

Comment:
It worth adding it into documentation with a link to the official Android documentation

Reply:
It's nice option to have, but I don't see a situation when I want to use it tbh

Reply:
Documentation PR here: https://github.com/defold/doc/pull/441

Reply:
Ah, yes. I forgot